packages/orbit-components/src/List/ListItem/index.tsx

```diff
@@ -12,7 +12,10 @@ const ListItem = ({ label, children, icon = <CircleSmall />, dataTest }: Props)
       className="orbit-list-item font-base mb-100 flex last:m-0 last-of-type:m-0"
     >
       {icon && (
-        <div className="orbit-list-item-icon me-200 [&>.orbit-carrier-logo]:size-icon-small [&>.orbit-carrier-logo>img]:size-icon-small flex flex-none">
+        <div
+          className="orbit-list-item-icon me-200 [&>.orbit-carrier-logo]:size-icon-small [&>.orbit-carrier-logo>img]:size-icon-small flex flex-none"
+          aria-hidden
```

Comment:
It should not have aria-hidden by default, should it? If anything, the icon on the default value for `icon` prop can have it. But it's devs' responsibility to add aria-hidden, if relevant.

Reply:
Actually, I believe this makes more sense than creating a new prop. I would just adjust the default value for the icon here (line 8) to `<CircleSmall ariaHidden />`.

Therefore, we don't need to add a new ariaHidden prop.

Reply:
I removed `aria-hidden` at all and put this attribute into `CircleSmall` icon directly.